cpp/src/io/orc/reader_impl_decode.cu

```diff
@@ -192,18 +192,14 @@ rmm::device_buffer decompress_stripe_data(
 
   // Dispatch batches of blocks to decompress
   if (num_compressed_blocks > 0) {
-    device_span<device_span<uint8_t const>> inflate_in_view{inflate_in.data(),
-                                                            num_compressed_blocks};
-    device_span<device_span<uint8_t>> inflate_out_view{inflate_out.data(), num_compressed_blocks};
```

Comment:
unrelated to header changes, just noticed that the spans are created for no reason here.